src/images/nubots-icon.svg

```diff
@@ -0,0 +1 @@
5-1.195 1.288-1.635 1.895-1.97.322-.176 3.032-.324 3.912-.234-.186-.374-.376-.679-.538-.834-.363-.35-.845-.519-2.085-.733-1.216-.21-1.666-.176-2.142.205-.323 1.232-.667 1.92-1.408 3.014-.889 1.309-1.52 2.19-2.055 2.82-.042.656.086 1.474.392 2.528.37 1.275.752 2.072 1.144 2.391zM10.58 17.59l-.083.075-1.539-.005c-1.289-.005-1.914-.08-2.297-.3a2.273 2.273 0 00-.48.139c-.515.252-.85.844-.97 1.715-.056.408-.175 1.218-.266 1.802-.137.893-.157 1.306.023 1.762.055-.509.836-2.888 1.171-3.551.09-.178.29-.445.445-.593.278-.267.298-.27 2.08-.257 1.402.011 1.867.05 2.107.18.385.21 1.032.907 1.62 1.747.248.356.736.98 1.085 1.387.616.719.634.757.633 1.347-.001.487-.103.848-.517 1.832-.533 1.265-.908 1.893-1.32 2.2.749-.223 1.373-.458 1.617-.608a.299.299 0 00.105-.09c.02-.04.059-.183.107-.388.062-.262.14-.627.212-1.006.43-2.248.19-2.895-2.047-5.506-.894-1.043-1.352-1.586-1.687-1.883zM22.415 20.675c-.456-.311-.892-.582-1.147-.698-.593-.27-.773-.229-1.864.432-.464.282-1.158.652-1.542.823-.787.35-1.571.9-1.803 1.265-.061.097-.16.425-.269.85.32-.079.787-.18 1.26-.275.762-.15 1.672-.41 2.072-.59.975-.442 1.952-.644 2.438-.506.669.191 1.884 1.599 2.595 2.995.035-.134.07-.272.106-.42.072-.299.13-.554.172-.778-.67-.74-1.82-2.384-1.96-2.864a2.613 2.613 0 01-.058-.234zM27.133 10.431c-.675-.496-1.03-.733-1.342-.817-.053 1.049-.278 1.383-1.08 1.99-.38.289-1.019.706-1.418.929-.657.365-1.07.476-1.642.415 0 .443-.006 1.055 0 1.815.018 1.729.075 3.315.13 3.525.084.326.3.614.69.91.008-.479.04-1.082.098-1.864.123-1.674.385-3.373.582-3.776.12-.248.98-.915 2.396-1.862.505-.339.87-.388 1.265-.17.311.172 1.707 1.847 2.311 2.772.096.147.179.283.25.411-.093-1.088-.284-2.157-.476-2.54-.247-.493-.759-.997-1.764-1.738z" fill="#F9A50D"/><path d="M29.299 24.299a15.768 15.768 0 01-2.78 3.848c-.182-.052-.43-.13-.682-.218-.829-.287-1.336-.324-2.024-.053-.696 1.423-1.058 1.718-2.513 2.227-.622.217-1.436.466-1.81.556-.24.057-.5.098-.758.123a18 18 0 00-.1.642 9.161 9.161 0 003.455-.979c1.12-.555 1.617-.668 2.147-.488l.103.034a14.709 14.709 0 01-8.327 2.667l-.133.002c-2.34 0-4.56-.551-6.542-1.534v-.007c-.018-.664-.416-2.11-.735-2.674-.374-.659-1.336-1.788-2.911-3.418a21.06 21.06 0 00-.938-.927c.094.147.186.294.27.433.333.555.836 1.29 1.118 1.63.983 1.19 1.07 1.42 1.07 2.817v.863a15.731 15.731 0 01-5.516-7.275l.473-.016-.397-.173-.172-.075a16.139 16.139 0 01-.912-5.365 16.214 16.214 0 01.961-5.504A15.733 15.733 0 017.954 3.53a14.726 14.726 0 018.883-2.28l.256.16c.338.213.803.46 1.033.55.09.033.242.092.429.164-.14-.181-.27-.366-.379-.543l-.124-.202a14.868 14.868 0 017.167 3.17c.51.413.995.86 1.45 1.336a15.82 15.82 0 013.154 4.82 4.57 4.57 0 01-.116.374c-.176.508-.173.534.125 1.739.362 1.457.462 3.226.26 4.576-.062.404-.09.84-.075 1.063.393-2.272.55-2.955.812-3.558l.086-.199a16.244 16.244 0 01-.066 4.912 15.968 15.968 0 01-1.55 4.687zM15.877.772C7.262.772.253 8.024.253 16.94c0 8.914 7.01 16.166 15.624 16.166 8.616 0 15.625-7.252 15.625-16.166 0-8.915-7.01-16.167-15.625-16.167z" fill="#F9A50D"/></svg>
```

Comment:
How come this has been added? I don't see a nubots icon in the contributors/last edited bar?

Reply:
It's used for authors without a GitHub profile, as the default avatar. E.g. if someone like Stephan (assuming he doesn't have a GH account) were to add to a page they can be credited with just their name, and the NUbots icon will be used, without linking to GitHub.

Reply:
Ah, ok.